src/gt4py/gtc/python/npir.py

```diff
@@ -47,8 +57,14 @@ def k(cls, offset: int, *, parallel=False) -> "AxisOffset":
         return cls.from_int(axis_name=AxisName.K, offset=offset, parallel=parallel)
 
 
-class VectorExpression(gtir.LocNode):
-    pass
+class VectorExpression(Expr):
+    # MARK: remove when abstract nodes implemented in eve
```

Comment:
Why `MARK`?

Reply:
because it might never get done but when it happens it is easy to search & replace for

Reply:
will change it to `TODO` to stick with the current convention.